### PR TITLE
feat(components): [image-viewer] add `minScale` and `maxScale`

### PR DESCRIPTION
**Pull request by IceyWu**
fix(components): [image-viewer] add scale props,minScale to control the minimum scale of the image viewer zoom event,maxScale to control the maximum scale of the image viewer zoom event.

### CODE REVIEW

**Review by btea — COMMENTED** [review]
These two properties should also be added to the document.

**Comment by IceyWu** [comment]
> These two properties should also be added to the document.

The properties of the image component in the document?

**Comment by btea** [comment]
https://element-plus.org/en-US/component/image.html#image-viewer-attributes

**Comment by IceyWu** [comment]
Done！😊

**Comment by IceyWu** [comment]
> https://element-plus.org/en-US/component/image.html#image-viewer-attributes

Already done and pull

**Comment by btea** [comment]
The image attribute should also be added. https://element-plus.org/en-US/component/image.html#image-attributes

**Review by btea — APPROVED** [review]
Thanks！

File: docs/en-US/component/image.md [hunk 1]
```diff
@@ -108,6 +110,8 @@ image/image-preview
 | hide-on-click-modal   | whether user can emit close event when clicking backdrop.                                                                     | ^[boolean]            | false   |
 | teleported            | whether to append image itself to body. A nested parent element attribute transform should have this attribute set to `true`. | ^[boolean]            | false   |
 | zoom-rate ^(2.2.27)   | the zoom rate of the image viewer zoom event.                                                                                 | ^[number]             | 1.2     |
+| min-scale ^(2.3.13)   | the min scale of the image viewer zoom event.                                                                                 | ^[number]             | 0.2     |
+| max-scale ^(2.3.13)   | the max scale of the image viewer zoom event.                                                                                 | ^[number]             | 7       |
```
Comment (tolking) [inline docs/en-US/component/image.md:114]:
```suggestion
| min-scale ^(2.4.0)   | the min scale of the image viewer zoom event.                                                                                 | ^[number]             | 0.2     |
| max-scale ^(2.4.0)   | the max scale of the image viewer zoom event.                                                                                 | ^[number]             | 7       |
```

File: docs/en-US/component/image.md [hunk 2]
```diff
@@ -76,6 +76,8 @@ image/image-preview
...
 | infinite              | whether the viewer preview is infinite.                                                                                                           | ^[boolean]                                                              | true    |
 | zoom-rate             | the zoom rate of the image viewer zoom event.                                                                                                     | ^[number]                                                               | 1.2     |
+| min-scale             | the min scale of the image viewer zoom event.                                                                                                     | ^[number]                                                               | 0.2     |
+| max-scale             | the max scale of the image viewer zoom event.                                                                                                     | ^[number]                                                               | 7       |
```
Comment (tolking) [inline docs/en-US/component/image.md:80]:
```suggestion
| min-scale ^(2.4.0)            | the min scale of the image viewer zoom event.                                                                                                     | ^[number]                                                               | 0.2     |
| max-scale ^(2.4.0)             | the max scale of the image viewer zoom event.                                                                                                     | ^[number]                                                               | 7       |
```

File: docs/en-US/component/image.md (hunk 1 above)
Comment (tolking) [reply docs/en-US/component/image.md:114]: @IceyWu Can you update the version?

File: docs/en-US/component/image.md (hunk 2 above)
Comment (IceyWu) [reply docs/en-US/component/image.md:80]: sure!

**Review by tolking — APPROVED** [review]
Thank for your contribution

**Pull request merged**